package.json

```diff
@@ -75,15 +76,38 @@
     "webpack-dev-server": "^3.11.0"
   },
   "devDependencies": {
+    "@linagora/karma-ng-jade2module-preprocessor": "0.5.3",
+    "angular-mocks": "1.3.20",
+    "chai": "3.0.0",
+    "chai-datetime": "1.4.0",
+    "chai-shallow-deep-equal": "1.4.0",
     "css-loader": "^3.6.0",
     "exports-loader": "^1.0.0",
     "expose-loader": "^1.0.0",
     "file-loader": "^6.0.0",
     "glob-all": "^3.2.1",
+    "html-loader": "^1.1.0",
     "html-webpack-plugin": "^4.3.0",
+    "ignore-loader": "^0.1.2",
+    "imports-loader": "^1.1.0",
+    "karma": "4.2.0",
+    "karma-coverage": "1.1.2",
+    "karma-mocha": "1.1.1",
+    "karma-chrome-launcher": "2.2.0",
+    "karma-firefox-launcher": "1.1.0",
+    "karma-rawfixtures-preprocessor": "linagora/karma-rawfixtures-preprocessor",
```

Comment:
Can you check if the tests actually need to use fixtures? If not, you can remove everything related to fixtures.

Reply:
ye it's useless